src/Microsoft.Health.Fhir.Shared.Api/Features/Operations/OperationsCapabilityProvider.cs

```diff
@@ -75,5 +84,36 @@ public void AddMemberMatchDetails(ListedCapabilityStatement capabilityStatement)
         {
             GetAndAddOperationDefinitionUriToCapabilityStatement(capabilityStatement, OperationsConstants.MemberMatch);
         }
+
+        public void AddPatientEverythingDetails(ListedCapabilityStatement capabilityStatement)
```

Comment:
We currently have methods like this in the OperationsCapabilityProvider files in the version specific packages so that we don't need to check for the FHIR version here. Could you move this method to those files?

Reply:
Sure, the methods are moved to version specific OperationsCapabilityProvider files in the update.